_sources/CSPPracticeQuestions/exam24_3.rst

```diff
@@ -72,10 +72,10 @@ You will not be able to change your answers after you hit the "Finish Exam" butt
        :answer_c: What is the average annual precipitation?
        :answer_d: What is the average daily temperature? 
        :correct: c
-       :feedback_a: Incorrect
-       :feedback_b: Incorrect
-       :feedback_c: Correct
-       :feedback_d: Incorrect
+       :feedback_a: Incorrect. The meteorologists have temperature data from day and nights.
+       :feedback_b: Incorrect. The meteorologists have temperature and precipitation data.
+       :feedback_c: Correct. The meteorologists only have data from one year, so they cannot calculate the average annual precipitation.
+       :feedback_d: Incorrect. The meteorologists have temperature data for an entire year.
```

Comment:
It would be nice to say something like "The meteorologists have temperature data for an entire year, so they can find the average temperature value over that time." Give a little more description.